client/components/date-range/style.scss

```diff
@@ -0,0 +1,129 @@
+.date-range {
+	position: relative;
+	display: flex;
+	align-items: baseline;
+}
+
+
+.date-range__trigger-btn {
+	text-align: left; // required to avoid content "jumping" as date text changes
+
+	.toggle-visible & {
+		box-shadow: none;
+		background-color: var(--color-neutral-0);
+	}
+
+	.gridicons-chevron-down {
+		fill: var(--color-neutral-500);
+		margin: 0 0 0 5px;
+		flex-shrink: 0;
+		transition: transform 0.15s cubic-bezier( 0.175, 0.885, 0.32, 1.275 );
+
+		.is-disabled & {
+			fill: var(--color-neutral-50);
+		}
+
+		.toggle-visible & {
+			transform: rotate( -180deg );
+		}
+	}
+}
+
+
+.date-range__trigger-btn-text {
+	width: 12.5em; // required to avoid content "jumping" as date text changes
+    display: inline-block;
+}
+
+.date-range__popover-inner {
+	padding: 15px;
+	max-width: 90%;
+
+	@include breakpoint( '>480px' ) {
+		width: 400px;
+	}
+}
+
+.date-range__popover-header {
+	.button {
+		margin: 0 0.5em;
+	}
+}
+
+.date-range__popover {
+	.DayPicker {
+		margin-top: 12px;
+		border-top: 1px solid var(--color-neutral-50);
+	}
+	
+	.DayPicker-wrapper,
+	.DayPicker-Months {
+		display: flex;
+	}
+
+	.DayPicker-Month {
```

Comment:
Just pointing this out so that we either make use of it or 🔪it out before merge :)

Reply:
Note to self: this refers to need to remove unused selector

Reply:
Fixed